src/Tribe/RSVP.php

```diff
@@ -375,32 +375,32 @@ public function update_attendee_data( $data, $order_id, $event_id ) {
 
 		// Get the Attendee Data, it's important for testing
 		foreach ( $rsvp_orders as $test_attendee ) {
-			if ( $order_id !== $test_attendee['order_id'] ) {
+			if ( $order_id !== $test_attendee[ 'order_id' ] ) {
 				continue;
 			}
 
 			$attendee = $test_attendee;
 		}
 
 		// Dont try to Save if it's restricted
-		if ( ! isset( $attendee['product_id'] )
-		     || $this->tickets_view->is_rsvp_restricted( $event_id, $attendee['product_id'] )
+		if ( ! isset( $attendee[ 'product_id' ] )
+		     || $this->tickets_view->is_rsvp_restricted( $event_id, $attendee[ 'product_id' ] )
 		) {
 			return;
 		}
 
-		$attendee_email = empty( $data['email'] ) ? null : sanitize_email( $data['email'] );
+		$attendee_email = empty( $data[ 'email' ] ) ? null : sanitize_email( $data[ 'email' ] );
 		$attendee_email = is_email( $attendee_email ) ? $attendee_email : null;
-		$attendee_full_name = empty( $data['full_name'] ) ? null : sanitize_text_field( $data['full_name'] );
-		$attendee_optout = empty( $data['optout'] ) ? false : (bool) $data['optout'];
+		$attendee_full_name = empty( $data[ 'full_name' ] ) ? null : sanitize_text_field( $data[ 'full_name' ] );
+		$attendee_optout = empty( $data[ 'optout' ] ) ? false : (bool) $data[ 'optout' ];
 
-		if ( empty( $data['order_status'] ) || ! $this->tickets_view->is_valid_rsvp_option( $data['order_status'] ) ) {
+		if ( empty( $data[ 'order_status' ] ) || ! $this->tickets_view->is_valid_rsvp_option( $data[ 'order_status' ] ) ) {
 			$attendee_order_status = null;
 		} else {
-			$attendee_order_status = $data['order_status'];
+			$attendee_order_status = $data[ 'order_status' ];
 		}
 
-		$product_id  = $attendee['product_id'];
+		$product_id  = $attendee[ 'product_id' ];
```

Comment:
no spaces around the array keys above unless the key name is a variable — `$attendee['product_id']`, `$data['order_status']`, etc. are fine here.